src/website/views/javascript/popover.pug

```diff
@@ -0,0 +1,297 @@
+---
+title: Popover
+---
+
+p.-text
+  | Popovers provide an contextual space, attached to an element and in an upper z-index layer, where you can
+  | include any kind of HTML code related to the trigger.
+
+h3 Base
+p.-text
+  | To render a popover, add your HTML code to a <code>data-popover-content</code> attribute. Then,
+  | you can initialize it with the <code>chi.popover</code> factory method.
+.example.-mb--3
+  .-p--3
+    button#popover-1(data-popover-content='<div class="-px--2"><h4>Popover!</h4><p>This is a popover content triggered by a button. </p></div>').a-btn.-mr--2 Popover
```

Comment:
Can we update all the popover examples to default to these styles:

- Use `h5` instead of `h4` for popover titles
- Change classes on the popover content container to `-px--2 -py--1 -s--3`
- Ensure popover `p` text renders with a 1.25rem (20px) line-height

![image](https://user-images.githubusercontent.com/130654/53534935-ab5cfb80-3ab5-11e9-81ed-9d6adaa39772.png)

Reply:
Done!